src/components/Home/Home.js

```diff
@@ -1,13 +1,83 @@
-import React from "react";
+import React, { useEffect, useState } from "react";
 import "components/Home/Home.scss";
 import SlateEditor from "components/SlateEditor/SlateEditor";
 import Sidebar from "components/Sidebar/Sidebar";
+import { useNavigate, useParams } from "react-router";
+import axios from "axios";
+import { decodeToken } from "react-jwt";
+import { CircularProgress } from "@mui/material";
 
 export default function Home() {
+  const navigate = useNavigate();
+  var { id } = useParams();
+  const [options, setOptions] = useState([]);
+  const [documentId, setdocumentId] = useState("");
+  const [loading, setLoading] = useState(false);
+
+  useEffect(() => {
+    async function documentsArray() {
+      try {
+        setLoading(true);
+        var userId = await decodeToken(localStorage.getItem("accessToken"))[
+          "user_id"
+        ];
+        var config = {
+          headers: {
+            "Content-Type": "application/json",
+            Authorization: `Bearer ${localStorage.getItem("accessToken")}`,
```

Comment:
aovid multiple calls to get localstorage.accesstoken; get once in the beginning, reuse